01-getting-started/installation/platforms/install-on-gcp.md

```diff
@@ -0,0 +1,195 @@
+---
+Description: >-
+  Learn how to quickly deploy Squidex (with MongoDB) on Google Cloud Platform using Cloud Run & MongoDB Atlas.
+---
+
+# Deploy Squidex on Google Cloud Platform using Cloud Run, Cloud Storage & MongoDB Atlas
+
+This version of the installation tutorial deploys Squidex as a container on [Cloud Run](https://cloud.google.com/run/) in Google Cloud Platform. The approach is PaaS or Managed service.
```

Comment:
What does this mean?

Reply:
Sorry, not sure what part you are referring to.

Reply:
Ah, okay. I have already edited this statement to make to more comprehensible.